### ISSUE TITLE
Treasure Table: 1 should be better than 2...

### ISSUE THREAD

**Issue by sarinilla** [ricegf/elven-fire#18]
Currently we have a tendency to mulligan our treasure rolls DOWN from a 1 to a 2 in order to get a greater artifact.  Instead, when rolling twice more, the lesser roll should automatically become a greater artifact.

**Comment by JaceR** [comment]
I disagree, I don't think it should automatically become Greater, but it should automatically become an Artifact. (Maybe that says the same thing, since Greater includes Lesser.) 

Could be interesting if in some way the higher levels of treasure tables made the lowest Lesser Artifacts no show up on the Greater Artifacts roll.

**Comment by ricegf** [comment]
The "1 is a 2 and a second roll" concept seems to be working fairly well IMHO - we had an unusually large number of examples last night.  :-)

I favor adopting it for the next version.

Jace, I originally made the Greater Artifact roll include the Lesser Artifact table to compensate for the paucity of digits on a d8.  If you roll a 3, you get Lesser.  If you roll 2, you have a 50/50 chance of a Greater or Lesser. This is mathematically equivalent to 2 is Greater, 2.5, 3 and 3.5 is Lesser - just didn't have the half rolls to use.  :-D  But using a D16, the table could read 1-2 Roll Again  3 Greater Artifact 4-6 Lesser Artifact, and so on.

I can rework the tables to avoid splitting Greater is you feel it's worth the effort.

**Comment by JaceR** [comment]
I'm not sure. It just "hurts" to roll Greater... and then end up with a 'Lesser' treasure. So I was wondering if there was another option. No worries, though, there are bigger fish to fry.

**Comment by ricegf** [comment]
Yes, I think the game is better off if we reduce disappointments. So I'll play with the math and see if we can avoid the "I got a greater... potion" syndrome...
